cloud_blobstore/__init__.py

```diff
@@ -249,6 +249,10 @@ class BlobStoreCredentialError(BlobStoreError):
     pass
 
 
+class BlobStoreReadError(BlobStoreError):
+    pass
```

Comment:
Can you document exactly what this means?  It seems like this is only timeout errors.

Reply:
Documentation added.